suite/src/__tests__/fast/data-feed-api.test.ts

```diff
@@ -77,7 +77,7 @@ describe.skip('Datafeed SSE Api Test', () => {
       const doc = await genesisCommit(ceramicNode1, modelInstanceDocumentMetadata, false)
       expectedEvents.add(doc.tip.toString())
 
-      await accumulator.waitForEvents(expectedEvents, 1000 * 60)
+      await accumulator.waitForEvents(expectedEvents, 2000 * 60)
```

Comment:
```suggestion
      await accumulator.waitForEvents(expectedEvents, 1000 * 60 * 2)
```

Reply:
total unimportant nitpick, but personally I find it easier to mentally parse where these numbers are coming from when written out like this (millis->seconds->minutes)